cfme/tests/automate/custom_button/__init__.py

```diff
@@ -117,6 +117,22 @@ def is_displayed(self):
         return self.submit.is_displayed and self.service_name.is_displayed
 
 
+class TextInputAutomateView(View):
+    """This is view comes on clicking custom button"""
+
+    title = Text("#explorer_title_text")
+    text_box1 = TextInput(id="text_box_1")
+    text_box2 = TextInput(id="text_box_2")
+    submit = Button("Submit")
+    cancel = Button("Cancel")
+
+    @property
+    def is_displayed(self):
+        # This is only for wait for view
+        return (self.submit.is_displayed and self.text_box1.is_displayed
+                and self.text_box2.is_displayed)
```

Comment:
Can we format it like : 

            return (self.submit.is_displayed and
                    self.text_box1.is_displayed and 
                    self.text_box2.is_displayed)